rocAL/CMakeLists.txt

```diff
@@ -389,6 +391,14 @@ if(${BUILD_ROCAL})
     else()
         message("-- ${Yellow}NOTE: rocAL built without LibTar - WebDataset reader will not be supported${ColourReset}")
     endif()
+
+    # rocJPEG
+    if("${BACKEND}" STREQUAL "HIP" AND HIP_FOUND AND ROCJPEG_FOUND)
```

Comment:
@fiona-gladwin Is rocJpeg HW decoder an optional feature or required feature? If its optional, can we add a CMake flag to enable HW decoder? I can only see #ENABLE_HIP guards around rocJpeg decoder functions. Does that mean rocJpeg is required to be installed always if HIP is present?

Reply:
Better to add a CMake flag like ENABLE_ROCJPEG_DECODE

Reply:
Done